src/workers/gifparsing.worker.js

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
I know this was mostly copied from that gtk2k repo, but this file really could use some documentation

Reply:
Are you telling me that
`    pos += 13 + +!!(gif[10] & 0x80) * Math.pow(2, (gif[10] & 0x07) + 1) * 3;`
isn't _self documenting_?

jk. I didn't read this code. I'd have to refer to http://www.onicos.com/staff/iz/formats/gif.html or something similar to understand it. @netpro2k ?

Reply:
This also seems like a good resource (but I haven't read it). http://commandlinefanatic.com/cgi-bin/showarticle.cgi?article=art011